src/libraries/System.Security.Cryptography.Pkcs/src/System.Security.Cryptography.Pkcs.csproj

```diff
@@ -640,9 +638,6 @@
     <PackageReference Include="System.Buffers"  Version="$(SystemBuffersVersion)" />
     <PackageReference Include="System.Memory" Version="$(SystemMemoryVersion)" />
     <ProjectReference Include="$(LibrariesProjectRoot)System.Runtime.CompilerServices.Unsafe\src\System.Runtime.CompilerServices.Unsafe.ilproj" />
-    <!-- Ref assembly for netstandard2.0 isn't live built anymore. -->
-    <PackageDownload Include="$(MSBuildProjectName)" Version="[$(SystemSecurityCryptographyPkcsVersion)]" />
-    <ResolvedMatchingContract Include="$(NuGetPackageRoot)$(MSBuildProjectName.ToLowerInvariant())\$(SystemSecurityCryptographyPkcsVersion)\ref\$(TargetFramework)\$(MSBuildProjectName).dll" />
```

Comment:
Nit: I was looking at when this was introduced and found https://github.com/dotnet/runtime/blob/01b7e73cd378145264a7cb7a09365b41ed42b240/src/libraries/System.Security.Cryptography.Xml/src/System.Security.Cryptography.Xml.csproj#L92

Might be an opportunity for further cleanup, removing SystemSecurityCryptographyPkcsVersion entirely.  Not necessarily as part of this PR.

Reply:
I only grabbed for "$(ProjectName).csproj" but not for "$(ProjectName)" which is why I missed this dependent project. Thanks for the great feedback. I submitted https://github.com/dotnet/runtime/pull/52239 to address this.